x-pack/plugins/kubernetes_security/public/components/count_widget/helpers.test.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { addResourceTypeToFilterQuery, numberFormatter } from './helpers';
+import { CountResult } from '../../../common/types/count';
+
+const TEST_DATA_ARRAY: CountResult[] = [
+  32 as unknown as CountResult,
```

Comment:
Would love to see more test cases to test how the function handles edge cases, (e.g 0, negative number, Infinity, NaN...)

Won't need the unknown casts here after the fix mentioned above